Microsoft.Toolkit.Uwp.SampleApp/SamplePages/MasterDetail/MasterDetailXaml.bind

```diff
@@ -0,0 +1,38 @@
+<Page x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.RadialGaugePage"
```

Comment:
Yep, I haven't finished the sample page yet, it still looks a little, ahem, basic.